### ISSUE TITLE
JAWS with Firefox does not announce content with ARIA role="math"

### ISSUE THREAD

**Issue by mitchellevan** [FreedomScientific/standards-support#800]
### Summary

1. Use JAWS with Firefox
2. Go to [Math inside a rich text editor - testing user agent support (CodePen)](https://codepen.io/TPG/full/GRrKdop)
3. Use JAWS arrow keys to locate math elements not inside the editor: examples 2, 3, and 4

### Expected result

JAWS should announce the math structure and content, and the fact that it is a math element.

### Actual result

JAWS skips the element and does not announce it.

### Example

[Math inside a rich text editor - testing user agent support (CodePen)](https://codepen.io/TPG/full/GRrKdop)

#### JAWS version and build number

JAWS Version 2024.2312.53
 
#### Operating System and version

Windows 10

#### Browser and version:

Firefox 121.0.1
